stories/Welcome.mdx

```diff
@@ -1,7 +1,158 @@
 
-<div className="doc_nav_links">
-  <p>[Application Setup](?path=/docs/about-application-setup--docs)</p>
-</div>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-lg mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-1/4 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1>What is Gridiron Survivor?</h1>
+      <p class="text-2xl text-pretty max-w-screen-lg mx-auto">
+        Gridiron Survivor is the ultimate survivor league app. Track your picks, see how you stack up against your competition, and hope to be crowned champion! Developed by a talented team of Junior Developers led by Senior Software Engineer Shashi Lo, Gridiron Survivor is the perfect app for any football fan looking to take their survivor league to the next level.
+      </p>
+    </section>
+    <section class="sponsors-section h-fit">
+      <h2 class="pb-8">Sponsors</h2>
+      <div class="sponsors-card-container flex flex-row flex-wrap justify-between mx-auto">
+        <a href="https://appwrite.io/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={appwriteLogo} />
+          <h3>Appwrite</h3>
+        </a>
+        <a href="https://blogrecorder.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={blogRecorderLogo} />
+          <h3>Blog Recorder</h3>
+        </a>
+        <a href="https://www.frontendmentor.io/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={frontendMentorLogo} />
+          <h3>Frontend Mentor</h3>
+        </a>
+        <a href="https://www.gitkraken.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={gitkrakenLogo} />
+          <h3>GitKraken</h3>
+        </a>
+        <a href="https://usepastel.com/" target="_blank" rel="noreferrer" class="sponsor-card">
+          <Image src={pastelLogo} />
+          <h3>Pastel</h3>
+        </a>
+      </div>
+    </section>
+    <section class="meet-the-team space-y-12">
+      <h2>Meet the Team</h2>
+      <section class="meet-the-team grid grid-cols-4 grid-rows-auto place-items-center mx-auto gap-x-20 gap-y-10">
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Alex Appleget</h4>
+            <p>Software Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/alexappleget" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/alex-appleget/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Richard Choi</h4>
```

Comment:
Don't think it is a bad idea, but would rather devote time elsewhere.

I feel this would would be better implemented upon refactoring the page in MDX and creating React Components for the team member "cards".

Reply:
Decided to not focus on this for now, even with the refactor to MDX.

Perhaps we can create another Issue for it to tackle at a later date?